CLAUDE.md

```
@@ -0,0 +1,185 @@
+- **`@next-core/brick-container`**: 构件容器,用于运行微应用
+- **`@next-core/brick-playground`**: 构件开发调试工具(脱离核心框架)
+
+#### 已废弃的 V2 包
+
+- `brick-dll`, `brick-kit`, `brick-utils`, `rollup-config-factory`, `webpack-config-factory`
+- `@next-libs/*` → 使用 `@next-shared/*` 替代
+- `@next-sdk/*` → 使用 `@next-api-sdk/*` 替代
+
+### 项目结构
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)